### PR TITLE
high charts wrapper and example component pie chart. no tests yet.

### PR DESCRIPTION
**Pull request by harellevy**

### CODE REVIEW

File: projects/ui-framework/src/lib/charts/chart/chart-core.ts [hunk 1]
```diff
@@ -39,7 +38,7 @@ export abstract class ChartCore implements AfterViewInit, OnInit, OnChanges {
     '#FF9655',
     '#FFF263',
     '#6AF9C4'];
-  @Input() height = 700;
+  @Input() height = 500;
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/charts/chart/chart-core.ts]: we should default to the same size as the home page pie charts, 150x150

File: projects/ui-framework/src/lib/charts/donut-chart-text/donut-chart-text.stories.ts [hunk 2]
```diff
@@ -16,9 +16,9 @@ const template = `
     [name]="name"
     [donutInnerSize]="donutInnerSize"
   >
-    <b-display-4>
+    <b-display-3 style="color: #9d9c9c;">
       {{text}}
-    </b-display-4>
+    </b-display-3>
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/charts/donut-chart-text/donut-chart-text.stories.ts]: nice!

File: projects/ui-framework/src/lib/charts/pie-chart/pie-chart.stories.ts [hunk 3]
```diff
@@ -47,16 +47,17 @@ const note = `
...
+  donut | boolean | make pie chart donut chart | false
+  showDataLabels | boolean | shows label in pie | false
+  legend | boolean | shows legend | false
+  colorPalette | string[] | color palette array | default array of colors
+  height | number | height of chart | 500
+  name | string | name of series | none
+  donutInnerSize | number | defining the inner white circle in a donut pie chart | 60
+  donutWidth | number | overrides donutInnerSize by applying width of donut instead inner circle width | none
+  title | string | title of chart | none
+  pointFormat | string | tooltip formatter | {series.name}: <b>{point.percentage:.1f}%</b>
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/charts/pie-chart/pie-chart.stories.ts]: i think a lot of the inputs can be merged into an interface,
so should be something like:
```
dataConfig: {
name: string
data: (type?)
pointFormat: string (default to ...)
}
pieConfig: {
donut: boolean (default=true)
showLabels: boolean (default=false)
showLegend: boolean (default=false)
height: number (default=150)
donutWidth: number (default 35)
title: string (default to null)
}
```

donutInnerSize can be removed if we are overriding it with donutWidth, we should keep one or the other

File: projects/ui-framework/src/lib/charts/chart/chart-core.ts (hunk 1 above)
Comment (harellevy) [reply projects/ui-framework/src/lib/charts/chart/chart-core.ts]: I changed default value for pie chart and not all charts.

File: projects/ui-framework/src/lib/charts/pie-chart/pie-chart.component.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  @Input() name: string;
+  @Input() donut = false;
+  @Input() donutInnerSize = 60;
+  @Input() donutWidth: number;
+  constructor(
+    public zone: NgZone,
+    public cdr: ChangeDetectorRef
+  ) {
+    super(zone, cdr);
+    this.height = 150;
```
Comment (harellevy) [inline projects/ui-framework/src/lib/charts/pie-chart/pie-chart.component.ts]: this is what i did instead. initialise pie chart with its own default height on contractor.

File: projects/ui-framework/src/lib/charts/pie-chart/pie-chart.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+import * as Highcharts from 'highcharts';
+
+export class MockNgZone extends NgZone {
+  run(fn: Function): any {
+    return fn();
+  }
+}
+
+
+fdescribe('PieChartComponent', () => {
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/charts/pie-chart/pie-chart.component.spec.ts]: remove

**Pull request merged**